### PR TITLE
global: master refactoring

### PR DESCRIPTION
**Pull request by bouzlibop**

### CODE REVIEW

File: .travis.invenio.cfg [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# along with Zenodo. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Travis-CI configuration."""
+import getpass
+
+CFG_BIBSCHED_PROCESS_USER = getpass.getuser()
```
Comment (lnielsen) [inline .travis.invenio.cfg]: I don't think you need
```
CFG_BIBSCHED_PROCESS_USER
CFG_EMAIL_BACKEND
```

File: AUTHORS.rst [hunk 2]
```diff
@@ -43,3 +43,5 @@ Contact us at `info@invenio-software.org
 - Samuele Kaplun <samuele.kaplun@cern.ch>
 - Tibor Simko <tibor.simko@cern.ch>
 - Yoan Blanc <yoan.blanc@cern.ch>
+- Adrian Pawel Baran <adrian.pawel.baran@cern.ch>
```
Comment (lnielsen) [inline AUTHORS.rst]: Add kostas as well

File: .travis.invenio.cfg [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+DEBUG = False
+SECRET_KEY = 'MY_SECRET'
+
+# Disable all automatic asset building - false is /usr/bin/false.
+ASSETS_AUTO_BUILD = False
+
+PACKAGES = [
+    'invenio.modules.*',
+    'invenio_groups',
+]
```
Comment (lnielsen) [inline .travis.invenio.cfg]: Perhaps add
```
PACKAGES_EXCLUDE = [

]

File: AUTHORS.rst (hunk 2 above)
Comment (bouzlibop) [inline AUTHORS.rst]: :+1:

File: .travis.invenio.cfg (hunk 3 above)
Comment (bouzlibop) [inline .travis.invenio.cfg]: :ok_hand:

File: .travis.invenio.cfg (hunk 1 above)
Comment (bouzlibop) [inline .travis.invenio.cfg]: Looking into [travis build](https://travis-ci.org/inveniosoftware/invenio-groups/jobs/62393660) I think `CFG_BIBSCHED_PROCESS_USER` is needed:
```
ERROR: You must run "webcoll" as the same user that runs your Apache server
process (seems to be "www-data").
You may want to do "sudo -u www-data webcoll ..." to do so.
If you think this is not right, please set CFG_BIBSCHED_PROCESS_USER
appropriately and rerun "inveniocfg --update-config-py".
```

**Comment by bouzlibop** [comment]
Travis CI builds:
:heavy_check_mark: [lowest](https://travis-ci.org/inveniosoftware/invenio-groups/jobs/62396733)
:heavy_check_mark: [devel](https://travis-ci.org/inveniosoftware/invenio-groups/jobs/62396736)
:heavy_multiplication_x: [release](https://travis-ci.org/inveniosoftware/invenio-groups/jobs/62396735)

File: invenio_groups/models.py [hunk 4]
```diff
@@ -0,0 +1,775 @@
...
+    """Determine if group is system managed."""
+
+    privacy_policy = db.Column(
+        ChoiceType(PRIVACY_POLICIES, impl=db.String(1)), nullable=False,
+        default=PrivacyPolicy.ADMINS,
+        info=dict(
+            label=_('Privacy Policy'),
+            description=dict([
+                (PrivacyPolicy.PUBLIC,
+                 _('Group membership is fully public.')),
```
Comment (lnielsen) [inline invenio_groups/models.py]: I think it's better that these descriptions are added on the ``PrivacyPolicy`` class.

File: invenio_groups/models.py [hunk 5]
```diff
@@ -0,0 +1,775 @@
...
+    )
+    """Policy for who can view the list of group members."""
+
+    subscription_policy = db.Column(
+        ChoiceType(SUBSCRIPTION_POLICIES, impl=db.String(1)), nullable=False,
+        default=SubscriptionPolicy.CLOSED,
+        info=dict(
+            label=_('Subscription Policy'),
+            description=dict([
+                (SubscriptionPolicy.OPEN,
```
Comment (lnielsen) [inline invenio_groups/models.py]: ditto here.

**Comment by lnielsen** [comment]
Merged via b63a003547f8b9494e9bfab7de00fcb7101714f7

**Pull request closed without merging**